student.rb

```diff
@@ -18,4 +11,8 @@ def classroom=(classroom)
   def play_hooky
     '¯(ツ)/¯'
   end
+
+  def to_s
+    "[Student]: ID: #{@id} name: #{@name}, age: #{@age}, has permission: #{@parent_permission}"
+  end
 end
```

Comment:
Kindly format the UI output for selecting a person and book so that it matches the illustration in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ).

Note the following:

- Each listed item has a number attached
- You have books showing for both person and books

Your UI:
![image](https://user-images.githubusercontent.com/20554583/216425252-f01ede14-9c45-4e58-8d9d-e60d4b4360af.png)


Illustration UI:
![image](https://user-images.githubusercontent.com/20554583/216425174-f44d373d-8bdf-4eea-91d2-b57f6ff5fe41.png)